styles/styles.css

```diff
@@ -21,20 +25,30 @@ html, body {
     background-repeat: no-repeat;
     background-size: cover;
     min-width: 100vw;
-    min-height: 100vh;
```

Comment:
rollback this option and save min-height (cause shadow-box height becomes bigged than container0